src/strands/experimental/bidirectional_streaming/agent/agent.py

```diff
@@ -166,12 +60,13 @@ def __init__(
         hooks: Optional[list[HookProvider]] = None,
         trace_attributes: Optional[Mapping[str, AttributeValue]] = None,
         description: Optional[str] = None,
+        **kwargs: Any,
     ):
-        """Initialize bidirectional agent with required model and optional configuration.
+        """Initialize bidirectional agent with flexible model support and extensible configuration.
```

Comment:
nit: Minor nit. I prefer shorter docstrings typically
```suggestion
        """Initialize bidirectional agent.
```

Reply:
accepted!